### ISSUE TITLE
Good find!

### ISSUE THREAD

**Issue by pfalcon** [mahmoudimus/prothon#1]
Good find and computer archeology work! Do you know if its author is on github? Is it @mark-hahn?

**Comment by mahmoudimus** [comment]
Hi @pfalcon ! I know :) I had to really sleuth around to find this thing. Yes, I believe it is. Though, I think it is the copyright of [Hahn Creative Applications](https://github.com/mahmoudimus/prothon/blob/431233ac1c7c7d03d5f1d6f018946aef4643b79e/CHANGES.txt#L153). I would be happy to transfer this repo to @mark-hahn if he's interested.

**Comment by mark-hahn** [comment]
Weird, I seem to remember prothon as a close relative of python I wrote that was prototype-based.  I'm not sure though.  prothon.org seems to be about about a herpes drug.  I'd change the copyright to MIT if I could find where I could do that.

What does that have to do with this repo?  I'm very confused.

**Comment by pfalcon** [comment]
@mahmoudimus:

> I had to really sleuth around to find this thing.

Yes, that's what I mean, I had to answer Google's "do you really mean that?" a few times before I even was able to find relevant info, and no specific info (archive.org can help with that) or sources. 

@mark-hahn:

Thanks for response!

> I seem to remember prothon as a close relative of python I wrote that was prototype-based. I'm not sure though.

I was mentioned Prothon on a Python mailing lists. I never heard of it, and proceeded to look up. That was pretty hard on its own, as mentioned above (a random google-searcher would just concluded "there was no such thing as *that* Prothon"), but when I finally reached info, turned out, it was a bustling civilization with "3400 messages on the mailing list messages and many Python developers helping decision-making in the process", which then transformed into another language abandoning 1st language's premises with insightful comments like "[the real-world systems] are class-based and implementing the simple prototype objects would be a waste of performance (an impedance mismatch).", then transformed into yet another language. Then nothing. Barely any remnants can be seen on the surface of Earth of all that bustling activity. [Late Bronze Age Collapse](https://en.wikipedia.org/wiki/Late_Bronze_Age_collapse).

I consider myself an aspiring adept of the (non)[Survivorship-bias](https://en.wikipedia.org/wiki/Survivorship_bias) theory. So, the above piqued my interest. I wondered what the guy who spurred all that activity does *now*. And what I see is an account full of JavaScript and no, no mentioning of all the previous activity above.

I have very selfish interest in being an adept of the Survivorship-bias theory - I maintain a Python dialect myself (that's how I recognize @mahmoudimus, for him being kind enough to return to my projects related to that more than once (which is the usual starring pattern here on github, so exceptions can be spotted sometimes with a naked eye ;-) )). And I'm scared like hell of doing something which I actually don't need. And what's better sign that something wasn't needed if your (virtual) home doesn't even house yellowish, bleached out, but the snapshots of this previous activity?

So, I do study older projects, trying to do things as differently as reasonable (starting with doing as little as possible in the first place, investing freed time into studying (hi)stories like this).

@mark-hahn: Thanks for response again. And I hope all that brings good memories. (And I apologize in advance if not. For the history being a harsh thing, and we all die anyway.)
